staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4031,10 +4031,14 @@ func describeCSINode(csi *storagev1.CSINode, events *corev1.EventList) (output s
 			w.Write(LEVEL_1, "Drivers:\n")
 			for _, driver := range csi.Spec.Drivers {
 				w.Write(LEVEL_2, "%s:\n", driver.Name)
-				w.Write(LEVEL_3, "Allocatables:\n")
-				w.Write(LEVEL_4, "Count:\t%d\n", *driver.Allocatable.Count)
 				w.Write(LEVEL_3, "Node ID:\t%s\n", driver.NodeID)
-				w.Write(LEVEL_3, "Topology Keys:\t%s\n", driver.TopologyKeys)
+				if driver.Allocatable.Count != nil {
```

Comment:
I realize this PR was already merged, but saw this and wanted to ask...

Is checking `driver.Allocatable.Count != nil` sufficient to prevent a nil pointer exception?

How about this instead?
```
if driver.Allocatable != nil && driver.Allocatable.Count != nil {
```

Also, it seems like there is not an existing unit test covering describe csinode, but it would be nice to have one.

Reply:
Thanks for your review. @brianpursley 
See:
https://github.com/kubernetes/kubernetes/blob/a256498ba4e651ca0a3741707b4f5579a77e60ef/staging/src/k8s.io/api/storage/v1/types.go#L412-L424
So there are two reasons not considering `driver.Allocatable != nil`:
1. struct `Allocatable` only have one filed `count`.
2. Code here only describes `driver.Allocatable.Count` but not `driver.Allocatable`.

But the test is really needed, do you want to submit this test coverage PR?

Reply:
Thanks for the reply.

I’m not sure I understand yet.

Can `driver.Allocatable` be nil, and if so, won’t line 4035 fail when it tries to reference `.Count`?  Can you check it?

Reply:
Ah, I get your point. It won't fail. A test code can be :
```
package main

import (
	"fmt"
)

type TestStruct struct {
	A   string
	Sub TestSubStruct
}

type TestSubStruct struct {
	B string
	C *string
}

func main() {
	Test := TestStruct{A: "test"}
	// Test.Sub and Test.Sub.Hobby are nil
	if Test.Sub.C == nil {
		fmt.Println("Works")
	}
}
```
Is this right?

Reply:
I get your point !  I'll correct this!

Reply:
hi @zhouya0 Was there a follow on PR to address this comment. Thanks!